include/taskolib/Sequence.h

```diff
@@ -401,6 +401,9 @@ class Sequence
     /// Return the tags associated with this sequence in alphabetical order.
     const std::vector<Tag>& get_tags() const noexcept { return tags_; }
 
+    /// Return true if automated execution is active otherwise false.
+    bool get_automation() const noexcept { return automation_; }
```

Comment:
I find "automation" a bit generic. Maybe "autostart" or "autorun" would describe better what this is meant for. We already have a property called "AUTORUN" in the doocs-taskomatlib for that, but we can also still change that.

Reply:
OK. I will rename it to _autorun_.